core/src/test/java/org/opensearch/sql/expression/operator/arthmetic/MathematicalFunctionTest.java

```diff
@@ -565,6 +565,58 @@ public void exp_missing_value() {
     assertTrue(exp.valueOf(valueEnv()).isMissing());
   }
 
+  /**
+   * Test expm1 with integer value.
+   */
+  @ParameterizedTest(name = "expm1({0})")
```

Comment:
we should add tests for _at least_ each of the ExprCoreTypes:
```
          .add(BYTE)
          .add(SHORT)
          .add(INTEGER)
          .add(LONG)
          .add(FLOAT)
          .add(DOUBLE)
```

Reply:
We may want to add a test in the file that ensures that  ExprCoreType.numberTypes() doesn't change (or if it does, someone needs to take a look at this test and updates the file with new test cases)